integration/bmp/status_cmd_test.go

```diff
@@ -12,6 +12,11 @@ import (
 var _ = Describe("`$bmp status` integration tests", func() {
```

Comment:
Yes. This is the bmp status integration test. bmp login is executed in BeforeEach.